.github/workflows/build.yml

```
@@ -128,6 +128,7 @@ jobs:
           PLATFORM_DEV_DEVANT_ASGARDEO_CLIENT_ID: ${{ secrets.PLATFORM_DEV_DEVANT_ASGARDEO_CLIENT_ID }}
 
       - name: Run Trivy vulnerability scanner
+        if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)